apps/chat/src/components/ToolsetEditor/form.ts

```diff
@@ -0,0 +1,76 @@
+import { notAllowedSymbols } from '@/src/utils/app/file';
+import { getNextDefaultName } from '@/src/utils/app/folders';
+
+import { ToolsetModel } from '@/src/types/toolsets';
+
+import { DEFAULT_TOOLSET_NAME } from '@/src/constants/default-ui-settings';
+
+import { ToolsetTransportType } from '@epam/ai-dial-shared';
+import { z } from 'zod';
```

Comment:
I suggest to use more clear naming here.
```suggestion
import { z as zodValidation } from 'zod';
```